src/test/test-path-util.c

```diff
@@ -201,15 +203,19 @@ static void test_path_equal_root(void) {
 
 static void test_find_executable_full(void) {
         char *p;
+        char delim = '/';
+        char *test_file_name;
+        int fd;
+        _cleanup_strv_free_ char **exec_search_paths = strv_new("/tmp", "/bin");
```

Comment:
Could you also add an additional bogus dir like `/meow` before `/tmp` so we can test that it successfully skips it?